src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -111,5 +111,15 @@
             <argument type="service" id="translator"/>
             <argument type="string">%validator.translation_domain%</argument>
         </service>
+
+        <!-- SecurityExtension -->
+        <service id="form.extension.security" class="Symfony\Component\Form\Extension\Security\SecurityExtension">
+            <argument type="service" id="security.user_password_encoder" on-invalid="ignore"/>
+            <tag name="form.type_extension" />
+        </service>
+        <service id="form.extension.security.type.security_password" class="Symfony\Component\Form\Extension\Security\Type\SecurityPasswordType">
+            <argument type="service" id="security.user_password_encoder" on-invalid="ignore"/>
```

Comment:
This class would be pointless without the required password encoder, I'd suggest be strict here and remove if the service definition if security isn't enabled. See `form.type_extension.form.validator` as sample.